images/ceph/Makefile

```diff
@@ -79,39 +80,41 @@ do.build: generate-csv-ceph-templates
 		$(TEMP);\
 	fi
 	@rm -fr $(TEMP)
-	@$(MAKE) -C ../.. crds # revert changes made to the crds.yaml file during the csv-gen sequence
 
-generate-csv-ceph-templates: $(OPERATOR_SDK) $(YQ)
-	@if [ ! "$(INCLUDE_CSV_TEMPLATES)" = "" ]; then\
-		if [ "$(GOARCH)" = amd64 ]; then\
-			BEFORE_GEN_CRD_SIZE=$$(wc -l < ../../cluster/examples/kubernetes/ceph/crds.yaml);\
-			$(MAKE) -C ../.. NO_OB_OBC_VOL_GEN=true MAX_DESC_LEN=0 crds;\
-			AFTER_GEN_CRD_SIZE=$$(wc -l < ../../cluster/examples/kubernetes/ceph/crds.yaml);\
-			if [ "$$BEFORE_GEN_CRD_SIZE" -le "$$AFTER_GEN_CRD_SIZE" ]; then\
-				echo "the new crd file must be smaller since the description fields were stripped!";\
-				echo "length before $$BEFORE_GEN_CRD_SIZE";\
-				echo "length after $$AFTER_GEN_CRD_SIZE";\
-				exit 1;\
-			fi;\
-		fi;\
-		../../cluster/olm/ceph/generate-rook-csv-templates.sh;\
+# generate CSV template files into the directory defined by the env var CSV_TEMPLATE_DIR
+# CSV_TEMPLATE_DIR will be created if it doesn't already exist
+generate-csv-templates: $(OPERATOR_SDK) $(YQ) ## Generate CSV templates for OLM into CSV_TEMPLATE_DIR
+	@if [[ -z "$(CSV_TEMPLATE_DIR)" ]]; then echo "CSV_TEMPLATE_DIR is not set"; exit 1; fi
```

Comment:
Does `@[ -z "$(CSV_TEMPLATE_DIR)" ] && "CSV_TEMPLATE_DIR is not set" && exit 1` work? Or maybe `@test -z "$(CSV_TEMPLATE_DIR)" && "CSV_TEMPLATE_DIR is not set" && exit 1`.

You don't need to address this. I'm just not a big fan of the one-liner `if`

Reply:
I tried a bunch of iterations of this and things like it with no luck. Keep getting
```
make: *** [generate-csv-templates] Error 1
```

Reply:
No worries then.